[CELEBORN-1240] Introduce Husky Configuration to Celeborn Web (#2250)
apache/incubator-celeborn: CONTRIBUTING.md

dev/reformat
```

> If you have changed code for Web UI, Run following command before you create a new pull request.

Reviewer: ```suggestion
Meanwhile, run some checks of web with changes of web module before you create a new pull request.
```

> ```shell

Reviewer: Could the frontend code format introduce util in `dev/reformat`?

Author: Now. When executing `mvn install`, the code will be formatted.

Reviewer: @tiny-dust, could you update the `reformat` shell to provide developer with the way that reformats the frontend code and update the shell here?

Author: ok
